app/src/main/java/com/example/bigowlapp/model/Notification.java

```diff
@@ -15,7 +15,7 @@
     private String type;
     private Timestamp timeRead;
     private Timestamp timeSend;
-    private Schedule.UserResponse senderResponse;
+    private UserScheduleResponse senderResponse;
```

Comment:
@KhangZhengNg I think `UserScheduleResponse` should just be called `UserResponse`. It used by a Schedule, but its not necessarily related to schedule, such as here, which is just a generic notification.

Reply:
@KhangZhengNg Also I think the exists variable can be removed, assuming its unused.